pyriemann/utils/distance.py

```diff
@@ -6,59 +6,70 @@
 from .base import logm, sqrtm
 
 
+def _recursive(fun, A, B, *args, **kwargs):
+    """Recursive fuction with two inputs."""
```

Comment:
```suggestion
    """Recursive function with two inputs."""
```
I would also check that `A.shape == B.shape` or maybe just `len(A) == len(B)`